iOS Client/Store/MBStore.swift

```diff
@@ -96,35 +129,60 @@ class MBStore: NSObject {
         performDownload(clientFunction: client.getArticlesWithCompletion, persistentContainer: persistentContainer, deserializeFunc: MBArticle.deserialize, completion: completion)
     }
     
-    private func loadDevotions(completion: @escaping ([MBDevotion]?, Error?) -> Void) {
-        
-        let manager = FileManager.default
-        guard let url = manager.urls(for: .documentDirectory, in: .userDomainMask).first as URL? else {
-            completion(nil, MBDeserializationError.contextInsertionError(msg: "Could not create Devotions path"))
-            return
-        }
-        let path = url.path.appending("/devotions")
-        let pathUrl = URL(fileURLWithPath: path)
-        if !FileManager.default.fileExists(atPath: path) {
-            client.getDevotionsWithCompletion { data, error in
-                if error == nil, let data = data {
-                    self.save(data, pathUrl) { error in
-                        if error != nil {
-                            print("Could not save devotions data to disk")
+    func markDevotionAsRead(date: String, completion: @escaping (Error?) -> Void) {
```

Comment:
I think this could be done in a simpler way. Why de-couple the devotions from the read dates? Whether or not a devotion is `read` is a property of that devotion. It should be part of our MBDevotion model. 

When we first download the devotions data, we shouldn't blindly save it. We should get it from the external source (whether that's the internet or a file in the app bundle), deserialize it into our Model Objects, which can have extra fields beyond what's in the JSON like `isRead` that will default to `false` and then serialize the array of our model devotions into a file at the save path. 

The only issue is if we want to make the apps reload the devotions from the external source for some reason. In that case, we would need a helper function to merge what's new with what we have, but that's better in my opinion than maintaining dates separately and dealing with two different devotion models